.github/workflows/unit-test.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
```

Comment:
Why CI gets renamed?

Reply:
that job is actually unit tests. CI is too broad for that